backend/protocol_rpc/transactions_parser.py

```diff
@@ -30,11 +30,18 @@ def decode_signed_transaction(raw_transaction) -> DecodedTransaction:
             if signed_transaction_as_dict["to"]
             else None
         )
+        value = signed_transaction_as_dict["value"]
+        data = (
```

Comment:
can `value` and `data` coexist in the same transaction? Maybe we should have an "either" type to have a more accurate representation

Reply:
they both will exist at the same time when we implement calling a contract method and sending value to it.